src/material/datepicker/calendar-body.scss

```diff
@@ -31,13 +32,24 @@ $calendar-range-end-body-cell-size:
   padding-right: $calendar-body-label-side-padding;
 }
 
-.mat-calendar-body-cell {
+.mat-calendar-body-cell-container {
+  display: table-cell;
   position: relative;
+  padding: 0;
```

Comment:
Doesn't this get overwritten by the `style.paddingTop` and `style.paddingBottom` bindings? Also the `display: table-cell` shouldn't be necessary since the element is a `td` already.